ci/uninstall-thin-edge_io/uninstall-thin-edge_io.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/sh
+
+set -e
+
+usage() {
+    cat <<EOF
+USAGE:
+    delete-thin-edge_io [COMMAND]
+    
+COMMANDS:
+    remove     Uninstall thin-edge.io with keeping configuration files
+    purge      Uninstall thin-edge.io and also remove configuration files  
+
+EOF
+}
+
+stop_a_service_if_running() {
+    status=$(sudo systemctl is-active "$1") && returncode=$? || returncode=$?
```

Comment:
```suggestion
    local status
    status=$(sudo systemctl is-active "$1") && returncode=$? || returncode=$?
```

And what about the returncode thing here? You're not using it anywhere...